cnxepub/html_parsers.py

```diff
@@ -18,17 +19,6 @@
     }
 
 
-def _squash_to_text(elm, remove_namespaces=False):
-    value = [elm.text or '']
-    for child in elm.getchildren():
-        value.append(etree.tostring(child).decode('utf-8').strip())
-        value.append(child.tail or '')
```

Comment:
Notice this was removed in the new version. In the unit-tests it was duplicating the trailing text.